### ISSUE TITLE
Is there any way to capture the Whiteclicks IPs and condense them into a .txt file or something like that?

### ISSUE THREAD

**Issue by lariguedes98** [dvygolov/YellowCloaker#49]
First of all, I tried to access https://your.domain/logs?password=yourpassword as suggested in the documentation, but it displays the error below. Should this happen? In all the installations I have made of YellowCloaker, they all remain this way.

![image](https://github.com/user-attachments/assets/c40d8f5a-af09-454c-8c24-e1991d3a99ce)

The log folder is properly with the Whiteclicks and Blackclicks and there are several clicks in both records.

![image](https://github.com/user-attachments/assets/8ca3aace-f60a-4c1f-8d13-99309d340440)

Is there any way to automatically capture the IP of the leads that arrive in Whiteclicks? And condense this information into a blacklist.txt automatically?

My intention is to be able to determine this blacklist.txt in this option below:

![image](https://github.com/user-attachments/assets/69de6a51-68be-4450-bf37-c7b465727b33)

Thanks for your attention, have a nice day! (:

**Comment by dvygolov** [comment]
Nope, right now there is no way to get all whiteclicks ips into some kind
of a text file. Neither is there any way to automatically blacklist those
ips. What's the purpose of this? If your traffic is already being filtered
properly using the settings that you made - why do you need to create a
blacklist? How are you going to use that blacklist? Maybe in some other
installations of the cloaker, right? But then, why just not make the same
settings? No clues.
